Formula/ccache.rb

```diff
@@ -4,7 +4,7 @@ class Ccache < Formula
   url "https://github.com/ccache/ccache/releases/download/v4.6.1/ccache-4.6.1.tar.xz"
   sha256 "e5d47bd3cbb504ada864124690e7c0d28ecb1f9aeac22a9976025aed9633f3d1"
   license "GPL-3.0-or-later"
-  revision 1
+  revision 2
```

Comment:
```suggestion
  revision 1
```
Probably not worth a revision bump to add some symlinks that are relatively niche.

Reply:
Thanks, pushed that change